docs/install/quickstart.md

```diff
@@ -16,7 +16,7 @@ RKE2 provides an installation script that is a convenient way to install it as a
 ```
 curl -sfL https://get.rke2.io | sh -
 ```
-This will install `rke2-server` and `rke2-agent` services onto your machine.
+This will install the `rke2-server` service and the `rke2` binary onto your machine.
```

Comment:
I had both services here because they do both get installed.

Do you think that’s just too much noise for a QuickStart?

Reply:
It looks like only the service of the respective node type is being installed, at least with the auto detecting yum method on centos7/rhel7

Reply:
Yeah, my bad. It’s both on tar

Reply:
It is both with tarball but for a quickstart I can see the argument for leaving out that detail.